.eslintrc

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Can you add documentation for the configuration into `docs/tools/eslint.md`?

Reply:
Yeah, as we don't have any other documentation yet, I have added it inside the `docs` folder.